### ISSUE TITLE
Install Required Modules and Gin Theme on Drupal

### ISSUE THREAD

**Issue by Alina-Khmilevska** [Alina-Khmilevska/UPSA-Drupal#3]
**## Description**
As part of our ongoing development for the UPSA website, we need to ensure that the necessary modules are installed and configured. Additionally, we have chosen the Gin theme for its modern, clean, and accessible interface. This issue covers the installation and initial setup of the required Drupal modules and the Gin theme.

**## Tasks**
- [x] Research the latest stable versions of the necessary modules and the Gin theme.
- [x] Install the Gin theme via Composer.
- [x] Install the list of required modules via Composer.
- [x] Enable the Gin theme and configure as the default administration theme.
- [x] Ensure that all modules are properly configured and do not conflict with each other.

**## Required Modules**
- Pathauto
- Admin Toolbar
- Token
- Webform
- (Include any other modules specific to the project needs)

**## Dependencies**
- Ensure that the server environment meets the requirements for the latest version of Drupal and the specified modules and themes.
- Verify that Composer is up to date and functioning correctly.

**## Acceptance Criteria**
- The Gin theme is installed and set as the default for the administration interface.
- All required modules are installed and enabled without errors.
- The site is tested to ensure that the theme and modules are functioning as expected.

**## Notes**
- Before starting this task, review the existing configurations to ensure compatibility with the new modules and theme.
- If any issues arise during the installation process, create a separate issue to address each specific problem.
